### DISCUSSION TITLE
Getting error "unable to serialize, circular reference is too complex to analyze" V[5]

### DISCUSSION THREAD

**Discussion by pk4all** [adonisjs/core#3371] [Help]
Hello Team, 

Getting error "unable to serialize, circular reference is too complex to analyze",
My query is. 
```
await Post.query({connection:name}).preload('user').where({createdBy:user?.id,status:2}).first()
```
If I am using single argument in where clause I am getting result but getting error with more than one argument.
What is wrong with me?

Thanks.

**Comment by spiropoulos94** [comment]
Any updates on this?
